staging/src/k8s.io/apimachinery/pkg/api/meta/meta.go

```diff
@@ -40,8 +40,6 @@ func CommonAccessor(obj interface{}) (metav1.Common, error) {
 	switch t := obj.(type) {
 	case List:
 		return t, nil
-	case metav1.ListInterface:
```

Comment:
Why doesn't static check like this?

Reply:
In https://github.com/kubernetes/kubernetes/blob/23ddc3a876c272caa8119fcf3023ab2ebd6a0035/staging/src/k8s.io/apimachinery/pkg/api/meta/interfaces.go#L33 `List` is defined as `type List metav1.ListInterface`, hence the issue with [type assertion](https://staticcheck.io/docs/checks#SA4020).